Makefile

```diff
@@ -314,7 +314,9 @@ test: gotestsum
 	grep -v "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/" | \
 	grep -v "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/migrations/" | \
 	grep -v "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/" | \
-	grep -v "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/metrics/providers.go"  \
+	grep -v "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/metrics/providers.go" | \
+	grep -v "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api/error.go" | \
+	grep -v "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api/metadata.go" \
 	> coverage.out
```

Comment:
Instead of using all those pipes, this whole `cat cover.out | grep ... | grep ... > coverage.out`, could be rewritten as:
```bash
grep -v -e "_moq.go" \
    -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/cmd/kas-fleet-manager/main.go" \
    -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/" \
    -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/migrations/" \
    -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/connector/"  \
    -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/metrics/providers.go" \
    -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api/error.go" \
    -e "github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api/metadata.go" \
cover.out > coverage.out
```

Reply:
Thanks @ziccardi, this is much cleaner :)